packages/core/src/scss/components/foundation/forms/_forms.scss

```diff
@@ -28,12 +28,98 @@
         }
     }
 
+    #checkoutShippingAddress.form-fieldset:has(.floating-form-field) {
+        margin-bottom: 0.4rem;
+    }
+
+    .floating-fieldset {
+        .form-fieldset {
+            margin-bottom: 0;
+        }
+    }
+
     .form-field {
         position: relative;
 
         &:last-child {
             margin-bottom: 0;
         }
+
+        .floating-input,
+        .floating-select {
+            background-color: $floating-label-bg-color;
+            font-size: $floating-label-initial-font-size;
+            height: $floating-label-input-height;
+            margin-bottom: $floating-label-margin-bottom;
+            padding-bottom: 0;
+            padding-left: $input-horizontalPadding;
+            padding-right: $input-horizontalPadding;
+            padding-top: $input-verticalPadding * 1.5;
+        }
+
+        .floating-input:not(:focus),
+        .floating-select:not(:focus),
+        .floating-textarea {
+            border-color: $floating-label-border-color;
+        }
+
+        .floating-textarea {
+            background-color: $floating-label-bg-color;
+            padding-top: $input-verticalPadding * 2.5;
+        }
+
+        .dropdown-arrow,
+        .floating-dropdown-arrow {
+            height: 6px;
```

Comment:
Do we have utility functions that we can use here?

Reply:
I removed this line after realising it was unnecessary to set `height` here.
Also, all hard-coded values have been moved to the `_settings.scss` file.
These hard-coded values are new design styles for floating labels.